client/components/Editors/Table/index.tsx

```diff
@@ -25,8 +26,31 @@ export default function TableEditor(props: TableEditorProps) {
     () => createColumns(schema, report, history, selection),
     [schema, report, history, selection]
   )
+  const [rowsPerPage, setRowsPerPage] = React.useState(20)
+
+  const [rowHeight] = React.useState(40)
+
+  React.useEffect(() => {
+    const debouncedHandleResize = debounce(function handleResize() {
+      resizeTable()
+    }, 300)
+    window.addEventListener('resize', debouncedHandleResize)
+
+    return () => {
+      window.removeEventListener('resize', debouncedHandleResize)
+    }
+  })
+
+  function resizeTable() {
+    const tableElement = document.querySelector('.InovuaReactDataGrid__column-layout')
```

Comment:
Yea, probably in React-world it is kind a hack so we might need a comment here explaining it